solutions/swb-reference/scripts/generateCognitoTokens.js

```diff
@@ -20,8 +20,14 @@ const config = yaml.load(
   fs.readFileSync(join(__dirname, `../integration-tests/config/${process.env.STAGE}.yaml`), 'utf8') // nosemgrep
 );
 
-const clientId = config.clientId;
-const userPoolId = config.userPoolId;
+const apiStackOutputs = JSON.parse(
+  fs.readFileSync(join(__dirname, `../src/config/${process.env.STAGE}.json`), 'utf8') // nosemgrep
+);
```

Comment:
could we try catch and show a friendly error in case they want to run this before setting up the API